examples/guides/dataviews-widgets/step-3.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dataviews & Widgets - Step 3</title>
+
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css' rel='stylesheet' />
+  
+    <link href="https://libs.cartocdn.com/airship-style/v2.4.0/airship.css" rel="stylesheet" >
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+
+  </head>
+
+  <body class="as-app-body as-app">
+
+    <as-responsive-content>
+
+        <!-- Left sidebar -->
+        <aside class="as-sidebar as-sidebar--left">
+            <div class="as-container">
+                <section class="as-box">
+                    <as-formula-widget
+                        id="widgetFormula"
+                        class="as-p--16"
+                        heading="GDP"
+                        description="Sum of GDP">
+                    </as-formula-widget>
+                </section>
+                <section class="as-box">
+                  <as-category-widget
+                    id="widgetCategory"
+                    heading="Countries by continent (on screen)"
+                    description="Number of countries by continent">
+                  </as-category-widget>
+                </section>
+                <section class="as-box">
+                    <as-histogram-widget
+                      id="widgetHistogram"
+                      heading="Countries by population"
+                      description="Distribution of countries by population">
```

Comment:
I would add the option `show-clear` on the widgets in order to allow users clear filters applied like the examples `examples/widgets`